src/biogeochem/CNPhenologyMod.F90

```diff
@@ -378,7 +378,11 @@ subroutine CNPhenology (bounds, num_soilc, filter_soilc, num_soilp, &
        ! all restart files having been generated with
        ! https://github.com/ESCOMP/CTSM/issues/1623 resolved, or we stop having a time
        ! step 0 (https://github.com/ESCOMP/CTSM/issues/925).
-       if (num_pcropp > 0 .and. .not. is_first_step()) then
+!KO       if (num_pcropp > 0 .and. .not. is_first_step()) then
+!KO
+       ! Unknown if all restart files have been generated with 1623 resolved but we no longer have a time step 0
+       if (num_pcropp > 0) then
```

Comment:
- [ ] Great, thanks for this change. I agree with this and we can remove the above BACKWARDS_COMPATIBILITY comment.

Reply:
Thanks, yes, I had a comment in #925 noting that I checked for those things:
"Reviewed the code for instances of nstep == 0, nstep == 1, nstep /= 0, nstep > 0, etc (with and without spaces), and is_first_step, and made changes as required."
There could be some combination I didn't think of, if you or anyone has one I'll re-check.
I've removed the BACKWARDS_COMPATIBILITY comment in my latest commit.

Reply:
Oops, sorry, I missed that in my skim back through that issue. Thank you!

Reply:
@olyson can we resolve this conversation?

Reply:
Yes.